server/influxdb_test.go

```diff
@@ -103,3 +108,10 @@ func (suite *InfluxDBTestSuite) TestStoreHostTelemetry() {
 	suite.NoError(err)
 	suite.Equal(",_result,0,agent_id_2,uuid2,aws,16,15.2,16,16000", strings.TrimSpace(result))
 }
+
+func init() {
+	viper.SetDefault("influxdb-org", "suse")
+	viper.SetDefault("influxdb-bucket", "test-bucket")
+	viper.SetDefault("influxdb-url", "http://localhost:8086")
+	viper.SetDefault("influxdb-token", "telemetry-token")
+}
```

Comment:
super nitpick: usually you'll find `init()` functions at the beginning of the file just after the imports

Reply:
Thanks, moved.